src/util/numeric.h

```diff
@@ -144,36 +125,17 @@ inline bool isInArea(v3s16 p, v3s16 d)
 	);
 }
 
-#define rangelim(d, min, max) ((d) < (min) ? (min) : ((d)>(max)?(max):(d)))
-#define myfloor(x) ((x) > 0.0 ? (int)(x) : (int)(x) - 1)
-
 inline v3s16 arealim(v3s16 p, s16 d)
 {
-	if(p.X < 0)
```

Comment:
In this case its even OK I think, because it appears in a long chain. Our code styles shouldn't stand in the way if something is obviously better. You can leave it I think.